opengrok-web/src/main/webapp/search.jsp

```diff
@@ -71,8 +71,6 @@ include file="projects.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
 
-    long starttime = System.currentTimeMillis();
```

Comment:
The build failed -> you probably will need to add this line back :)

Reply:
forgot to remove all references to `starttime`, should be fixed in next build.